io/js/src/main/scala/fs2/io/file/FilesPlatform.scala

```diff
@@ -123,8 +123,8 @@ private[fs2] trait FilesCompanionPlatform {
         permissions: Option[Permissions]
     ): F[Path] =
       for {
-        dir <- createTempDirectory(dir, prefix, permissions)
-        path = dir / Option(suffix).filter(_.nonEmpty).getOrElse(".tmp")
+        dir <- createTempDirectory(dir, "", permissions)
+        path = dir / (prefix + suffix)
```

Comment:
Won't, in this way, all files have the same name? (Despite having different paths)

Reply:
Yes, but I don't think that's a problem, since the full path is unique.